gravitee-apim-gateway/gravitee-apim-gateway-core/src/test/java/io/gravitee/gateway/jupiter/core/v4/entrypoint/HttpEntrypointConnectorResolverTest.java

```diff
@@ -140,7 +140,7 @@ private Api buildApi() {
     }
 
     private Listener buildListener() {
-        final ListenerHttp listenerHttp = new ListenerHttp();
+        final HttpListener listenerHttp = new HttpListener();
```

Comment:
should we also rename variables ?
```suggestion
        final HttpListener httpListener = new HttpListener();
```